guides/release/components/introducing-components.md

```diff
@@ -166,9 +166,7 @@ A _component_ is kind of like your own custom HTML tag. You can tell that a tag
     <div class="cta-note-body">
       <div class="cta-note-heading">Zoey says...</div>
       <div class="cta-note-message">
-        A component's name is the same as its name on the file system,
-        capitalizing the first letter and every letter following a `-`, and
-        removing the `-` ("pascal case").
+        We capitalize the first letter and every letter after <code>-</code>, then remove the hyphens.
```

Comment:
I took the lines from https://github.com/ember-learn/guides-source/pull/1514/files#diff-f2b59cf002d648b76015ee7e339be532R232-R234
```suggestion
        A component's name is derived from its file name.
        We capitalize the first letter and every letter after <code>-</code>, then remove the hyphens.
        This is known as pascal case.
```